library/geonetwork

```diff
@@ -1,13 +1,22 @@
-# this file is generated via https://github.com/geonetwork/docker-geonetwork/blob/40ed4dce1cba141b69baffd2eac833e7f59d12aa/generate-stackbrew-library.sh
+# this file is generated via https://github.com/geonetwork/docker-geonetwork/blob/27a30d076d7358917d9f0d06577ff856f2e8e2b6/generate-stackbrew-library.sh
 
-Maintainers: Joana Simoes <jo@doublebyte.net> (@doublebyte1)
+Maintainers: Joana Simoes <jo@doublebyte.net> (@doublebyte1),
+     Juan Luis Rodriguez <juanluisrp@geocat.net> (@juanluisrp)
```

Comment:
@doublebyte1 I'm going to assume that since @juanluisrp has commit bits at https://github.com/geonetwork/docker-geonetwork that this is OK with you :smile: